.github/workflows/e2e-fixture-test.yml

```
@@ -0,0 +1,33 @@
+name: E2E Fixture Test
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - '**/*.md'
+      - 'docs/**'
+      - '.gitignore'
+      - 'LICENSE'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Dependencies
+      run: |
+        "${GITHUB_WORKSPACE}/hack/e2e/ensure_kubectl.sh"
+        "${GITHUB_WORKSPACE}/hack/e2e/ensure_cmctl.sh"
+
+    - name: Build BMO e2e Docker Image
+      run: |
+        export IMG=quay.io/metal3-io/baremetal-operator:e2e
+        make docker
```

I'm not sure how big difference this is really, but I think it is more common to set the environment this way in github actions.
```suggestion
    - name: Build BMO e2e Docker Image
      env:
        IMG: quay.io/metal3-io/baremetal-operator:e2e
      run: make docker
```